.github/workflows/canary-build.yml

```diff
@@ -0,0 +1,63 @@
+name: Canary build
+
+# Controls when the action will run. Triggers the workflow on push
+# events but only for the canary branch
+on:
+  push:
+    branches:
+      - canary
+
+# Publishes Canary builds with the dist tag of "canary"
+jobs:
+  # This workflow contains a single job called "publish"
+  publish:
+    name: Canary build publish
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - name: Checkout repo
+        uses: actions/checkout@v2
+
+      # Caches NPM files
+      - name: Cache NPM files
+        uses: actions/cache@v2
+        with:
+          path: ~/.npm
+          key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-node-
+
+      # Sets up Node v12
+      - name: Setup Node
+        uses: actions/setup-node@v1
+        with:
+          node-version: '12'
+          registry-url: 'https://npm.pkg.github.com'
+
+      # NPM ci init
+      - run: npm ci
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_PERSONAL_AUTH_TOKEN }}
```

Comment:
using a different auth token for NODE_AUTH_TOKEN? I'd expect it to be the same one, no? also, if you want, you can declare this at the top scope. There's an example of this in the canary publish workflow in blocks publish

Reply:
i saw that we were using two secrets on the blocks repo so i based it off that https://github.com/WPMedia/fusion-news-theme-blocks/blob/stable/.github/workflows/canary-build.yml#L31

Reply:
oh gotcha huh. wonder what the differences were